src/main/webapp/style.css

```diff
@@ -9,3 +9,7 @@
   border:5px solid black;
   line-height:50px;
 }
+
+#fail-message {
```

Comment:
Are other fail messages also going to be in red? if so, change this to a class instead of an id; ids are unique but classes can be used in many cases